### PR TITLE
Create aws account gitlab docs

### PR DESCRIPTION
**Pull request by yarden-port**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.

## Added docs pages

Please also include the path for the added docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

## Updated docs pages

Please also include the path for the updated docs

- Quickstart (`/`)
- Blueprint (`/platform-overview/port-components/blueprint`)
- ...

### CODE REVIEW

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples/create-new-aws-account-gitlab.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+}
+provider "aws" {
+  region = var.region
+}
+resource "aws_organizations_account" "account" {
+  name  = var.name
+  email = var.email
+  role_name = var.role_name
+  close_on_deletion = true
+  lifecycle {
```
Comment (stavbernazport) [inline docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples/create-new-aws-account-gitlab.md:146]: Could you explain why are you ignoring changes to the role_name here?

**Review by stavbernazport — COMMENTED** [review]
Small question
Looks good other than that

**Review by stavbernazport — APPROVED** [review]

File: docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples/create-new-aws-account-gitlab.md (hunk 1 above)
Comment (stavbernazport) [reply docs/build-your-software-catalog/sync-data-to-catalog/cloud-providers/aws/examples/create-new-aws-account-gitlab.md:146]: We talked about it

**Comment by stavbernazport** [comment]
LGTM, @hadar-co should you take it from here?

**Review by hadar-co — APPROVED** [review]

**Pull request merged**